finish/pom.xml

```diff
@@ -4,66 +4,40 @@
 
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>net.wasdev.wlp.maven.parent</groupId>
-        <artifactId>liberty-maven-app-parent</artifactId>
-        <version>RELEASE</version>
-    </parent>
-
     <groupId>io.openliberty.guides</groupId>
     <artifactId>guide-security-intro</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0-SNAPSHOT</version>
     <packaging>war</packaging>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <app.name>${project.artifactId}</app.name>
-        <testServerHttpPort>9080</testServerHttpPort>
-        <testServerHttpsPort>9443</testServerHttpsPort>
-        <package.file>${project.build.directory}/${app.name}.zip</package.file>
-        <packaging.type>usr</packaging.type>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <!-- Plugin versions -->
+        <version.liberty-maven-plugin>3.1</version.liberty-maven-plugin>
+        <version.maven-failsafe-plugin>2.22.2</version.maven-failsafe-plugin>
+        <version.maven-surefire-plugin>2.22.2</version.maven-surefire-plugin>
+        <!-- Liberty configuration -->
+        <liberty.var.default.http.port>9080</liberty.var.default.http.port>
+        <liberty.var.default.https.port>9443</liberty.var.default.https.port>
+        <liberty.var.application.name>${project.artifactId}.war</liberty.var.application.name>
```

Comment:
Why is this config needed?

Reply:
In the `server.xml` the following `${application.name}` is being used:
```
<application location="${application.name}"
               type="war"	
               id="${application.name}"
               name="${application.name}" context-root="/"> 
```
So the configuration `liberty.var.application.name` is placed in the `pom.xml`

Reply:
The `server.xml` has a variable `<variable name="application.name" defaultValue="guide-security-intro.war"/>` defined, that's being used in the `<application location..`. 

Looks like the `pom.xml` config is there to allow user to be able to modify that value.